src/components/Reports/ReportEdit.tsx

```diff
@@ -58,6 +58,15 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
 
   const [validateFields, setValidateFields] = useState(false);
 
+  const [idInView, setIdInView] = useState('');
+  useEffect(() => {
+    const el = document.getElementById(idInView);
+    if (el) {
+      el.scrollIntoView({ behavior: 'smooth' });
+      setIdInView('');
```

Comment:
reset the `idInView` so that if the same field errors it will get scrolled to again. Otherwise this effect isn't triggered.